CMakeLists.txt

```diff
@@ -210,9 +210,9 @@ INCLUDE_DIRECTORIES( ${ZEROMQ_INCLUDE_DIR} )
 SET(CMAKE_LIBRARY_PATH ${CMAKE_LIBRARY_PATH} ${ZEROMQ_ROOT}/lib)
 IF( WIN32 )
    IF(CMAKE_BUILD_TYPE STREQUAL "Debug")
-      SET( ZMQ_LIB_NAME "libzmq-v140-mt-gd-4_3_2" )
+      SET( ZMQ_LIB_NAME "libzmq-v142-mt-gd-4_3_2" )
```

Comment:
This will break Jenkins build

Reply:
Does it use VS2015? VS2017 should be detected by v141 check below, AFAIU.

Reply:
You right. VS2017 is used there. Also it is already stated as default VS version to use in our build instruction.

Reply:
Actually it will work only on old CMake. CMake 3.18 actually support [REQUIRED] for FIND_LIBRARY. So if zmq_v142 not found - cmake generation will stop with error.